### ISSUE TITLE
why second request Throw out Error: write after end In `watch-server`

### ISSUE THREAD

**Issue by Lanseria** [javieraviles/node-typescript-koa-rest#30]
```
  Error: write after end
      at writeAfterEnd (D:\Code\Github\demo-ts\node_modules\readable-stream\lib\_stream_writable.js:288:12)
      at PassThrough.Writable.write (D:\Code\Github\demo-ts\node_modules\readable-stream\lib\_stream_writable.js:332:20)
      at File.log (D:\Code\Github\demo-ts\node_modules\winston\lib\winston\transports\file.js:158:34)
      at File._write (D:\Code\Github\demo-ts\node_modules\winston-transport\index.js:81:19)
      at doWrite (D:\Code\Github\demo-ts\node_modules\readable-stream\lib\_stream_writable.js:428:64)
      at writeOrBuffer (D:\Code\Github\demo-ts\node_modules\readable-stream\lib\_stream_writable.js:417:5)
      at File.Writable.write (D:\Code\Github\demo-ts\node_modules\readable-stream\lib\_stream_writable.js:334:11)
      at DerivedLogger.ondata (D:\Code\Github\demo-ts\node_modules\readable-stream\lib\_stream_readable.js:619:20)
      at emitOne (events.js:121:20)
      at DerivedLogger.emit (events.js:211:7)
```
I only open the db logger
```
createConnection({
    type: 'postgres',
    host: connectionOptions.host,
    port: connectionOptions.port,
    username: connectionOptions.user,
    password: connectionOptions.password,
    database: connectionOptions.database,
    synchronize: true,
    logging: false,
    entities: [
       'dist/entity/**/*.js'
    ],
    extra: {
        ssl: config.dbsslconn, // if not development, will use SSL
    }
 })
```

**Comment by Lanseria** [comment]
I think error is here
```
//
// - Write all logs error (and below) to `error.log`.
// new winston.transports.File({ filename: 'error.log', level: 'info' })
```

**Comment by javieraviles** [comment]
You are so right, I have a private application in production using the boilerplate and yesterday I also realised of this...

The error was when rewriting in the error.log file, as it was already closed by the time it was trying to write into it the new log line. I think the error started happening a couple of weeks ago when I updated Winston to 3.0.0 and had to modified a bit the logging.ts file... I have fixed it together with a small fix in the new release 1.4.1.

In addition, the log level was 'info' when is should be 'error', also modified it.

Thanks for catching that :)
